.changes/nextrelease/bump-minimum-crt.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "type": "bugfix",
+    "category": "enhancement",
```

Comment:
I'd just leave this blank and maybe change the type to enhancement? I could go either way on the type though